src/dune_rules/bootstrap_info.ml

```diff
@@ -1,13 +1,14 @@
 open Import
-open Memo.O
 
 let def name dyn =
   let open Pp.O in
   Pp.box ~indent:2 (Pp.textf "let %s = " name ++ Dyn.pp dyn)
 
-let rule sctx ~requires_link (exes : Dune_file.Executables.t) () =
+let rule sctx ~requires_link (exes : Dune_file.Executables.t) =
+  let open Action_builder.O in
+  let* () = Action_builder.return () in
```

Comment:
What is the point of this?

Reply:
The old code had a `Memo.return () >>=` to make sure everything is delayed. I just stuck with it, although it doesn't change anything here.